.github/workflows/regen_certs.yml

```diff
@@ -0,0 +1,40 @@
+name: Regenerate certifactes
+on:
+  schedule:
+    # Run at 06:00 UTC, every Monday and Thursday.
+    - cron: '0 6 * * 1,4'
+jobs:
+  update:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13.1'
```

Comment:
Any particular reason to choose this package other than this was in the example? If not, let's try the latest stable 1.14.4.

Reply:
No longer relevant as have dropped the signed-exchange regen due to https://github.com/web-platform-tests/wpt/issues/24180#issuecomment-650947845